===================================================================
RCS file: /var/lib/cvs/sources/hurd/hurd/pfinet/ethernet.c,v
retrieving revision 1.29
retrieving revision 1.30
diff -u -r1.29 -r1.30
--- hurd/hurd/pfinet/ethernet.c	2002/05/29 08:34:35	1.29
+++ hurd/hurd/pfinet/ethernet.c	2006/04/27 00:10:51	1.30
@@ -68,6 +68,11 @@
 
 static short ether_filter[] =
 {
+#ifdef NETF_IN
+  /* We have to tell the packet filtering code that we're interested in
+     incoming packets.  */
+  NETF_IN, /* Header.  */
+#endif
   NETF_PUSHLIT | NETF_NOP,
   1
 };
